atlas/pixi.toml

```diff
@@ -0,0 +1,19 @@
+[project]
+name = "atlas-integration-challenge"
+version = "0.0.1"
+description = "IRIS-HEP ATLAS integration challenge"
+authors = ["Alexander Held <alexander.held@cern.ch>"]
+channels = ["conda-forge"]
+platforms = ["osx-arm64"]
+
+[tasks]
+
+[dependencies]
+coffea = ">=2025.7.3,<2026"
+jupyter = ">=1.1.1,<2"
```

Comment:
In general I replace the `jupyter` metapackage with `notebook` and `jupyterlab` to give me more control as a user on if I want the latest versions or not. The metapackage just gives you basically everything and is not really updated (4 times in like 10 years).

Reply:
That's good to know! Sure, we can switch to this.